.github/workflows/deploy-zeit-staging.yml

```diff
@@ -196,7 +220,7 @@ jobs:
         id: lighthouseCheck
         with: # See https://github.com/marketplace/actions/lighthouse-check#inputs for all options
           accessToken: ${{ secrets.GITHUB_CI_PR_COMMENT }} # Providing a token to comment the PR.
```

Comment:
`accessToken` isn't set in the production file, why?

Reply:
Because we were using `accessToken` only to comment a PR, not our workflow in production. I'm just going to comment it in staging because we don't use it anymore.